source/setup.Rmd

```diff
@@ -1,282 +1,277 @@
+**Configuring JupyterLab Desktop** From this point onward, with JupyterLab Desktop running,
+follow the instructions in the Ubuntu section on *Configuring JupyterLab Desktop* to set up the
+environment, install the JupyterLab Git extension, and install
+the various R software packages needed for the worksheets.
 
-**All operating systems:** 
-To install LaTeX, open JupyterLab by typing `jupyter lab` 
-in the Anaconda Prompt (Windows) or terminal (MacOS and Ubuntu) and press Enter.
-Then from JupyterLab, open an R console, type the commands listed below, and 
-press Shift + Enter to install `tinytex`:
 
-```
-tinytex::install_tinytex()
-tinytex::tlmgr_install(c("eurosym", 
-                         "adjustbox",
-                         "caption",
-                         "collectbox",
-                         "enumitem",
-                         "environ",
-                         "fp",
-                         "jknapltx",
-                         "ms",
-                         "oberdiek",
-                         "parskip",
-                         "pgf",
-                         "rsfs",
-                         "tcolorbox",
-                         "titling",
-                         "trimspaces",
-                         "ucs",
-                         "ulem",
-                         "upquote"))
-```
+### MacOS
 
-**Ubuntu:** 
-To append the TinyTex executables to our `PATH` we need to edit our `.bashrc file`. 
-The TinyTex executables are usually installed in `~/bin`. 
-Thus, add the lines below to the bottom of your `.bashrc` file 
-(which you can open by `nano ~/.bashrc` and save the file:
+**Installation** First, we will install Git for version control. 
+Open the terminal ([how-to video](https://youtu.be/5AJbWEWwnbY)) 
+and type the following command:
 
 ```
-# Append TinyTex executables to the path
-export PATH="$PATH:~/bin"
+xcode-select --install
 ```
+Next, visit the ["Installation" section of the JupyterLab Desktop homepage](https://github.com/jupyterlab/jupyterlab-desktop#installation).
+Download the `JupyterLab-Setup-MacOS-x64.dmg` or `JupyterLab-Setup-MacOS-arm64.dmg` installer file. 
+To know which installer is right for your machine, you need to know 
+whether your computer has an Intel processor (older machines) or an
+Apple processor (newer machines); the [Apple support page](https://support.apple.com/en-ca/HT211814) has
+information to help you determine which processor you have.
+Once downloaded, double-click the file to open the installer, then drag
+the JupyterLab Desktop icon to the Applications folder.  Double-click
+the icon in the Applications folder to start JupyterLab Desktop.
+
+**Configuring JupyterLab Desktop** From this point onward, with JupyterLab Desktop running,
+follow the instructions in the Ubuntu section on *Configuring JupyterLab Desktop* to set up the
+environment, install the JupyterLab Git extension, and install
+the various R software packages needed for the worksheets.
```

Comment:
I think organzation-wise, it might be best to have a section called "All operating systems"? For common steps? It is hard to refer back to where exactly the students should go to, in particular if we are not using headers that can be linked to.

Reply:
I tried to do this originally but struggled to add it without making a mess. I'll try again...no guarantees!

Reply:
Nope, it makes it a mess (it just means that both windows and mac users have to scroll around to find things, instead of just one of them). I'm going to change to your suggested Windows, Mac, Ubuntu order, but keep these in their own sections.